benchexec/tools/witnesslint.py

```diff
@@ -0,0 +1,21 @@
+import benchexec.result as result
+import benchexec.tools.template
+
+
+class Tool(benchexec.tools.template.BaseTool2):
+    """
+    Tool info for the witness checker (witnesslint)
+    (https://github.com/sosy-lab/sv-witnesses)
+    """
+
+    def executable(self, tool_locator):
+        return tool_locator.find_executable("witnesslint.py")
+
+    def name(self):
+        return "witnesslint"
```

Comment:
If the tool has a nicer human-readable name, you can return that here. Or keep it as it is.

Reply:
There's no other name for the tool. Should be fine.